python-3.12.yaml

```diff
@@ -0,0 +1,106 @@
+package:
+  name: python-3.12
+  # When bumping to a real non-prerelease you'll have to change the URL pattern on line 45
+  version: 3.12.0a5
+  epoch: 5
+  description: "the Python programming language"
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation: TODO
+      license: PSF-2.0
+  dependencies:
+    provides:
+      - python3=3.12.999
```

Comment:
Will this break 3.11 as the default?

Reply:
Yes! Good catch. I thought we had a system-python alias somewhere for the default but I don't see it.